### ISSUE TITLE
"Add to Home Screen" banner does not show up

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [Polymer/docs#1695]
I've visited https://www.polymer-project.org at least twice, with at least five minutes between visits, and Chrome does not automatically display the "Add to Home Screen" banner.

**Comment by FluorescentHallucinogen** [comment]
See https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/web-app-install-banners for more info.

**Comment by ebidel** [comment]
This heuristics aren't guaranteed to trigger it. If you open the dev tools and trigger it manually does it pop up?

**Comment by FluorescentHallucinogen** [comment]
I've disabled the visit frequency check by enabling `#bypass-app-banner-engagement-checks` flag on Chrome for Android. The banner does not show up.

I've also enabled `#enable-add-to-shelf` flag to test on desktop Chrome. The banner does not show up too.

Just checked https://guitar-tuner.appspot.com. All work like a charm!

Has https://www.polymer-project.org a registered service worker? Seems it's important criteria for Chrome to automatically display the banner.
